src/main/java/cloud/eppo/AppDetails.java

```diff
@@ -1,25 +1,25 @@
-package com.eppo.sdk.helpers;
+package cloud.eppo;
```

Comment:
on a conceptual level, is the `android` SDK a sub-implementation of the java SDK or are the `android` **and** `java-server` SDKs both parallel flavours of a `jvm` SDK?

Reply:
both are separate, completely individual libraries. they can share the same package names though, AFAIK there are no constraints preventing that